core/src/main/scala/kafka/log/Log.scala

```diff
@@ -2332,7 +2336,8 @@ object Log extends Logging {
           config,
           scheduler,
           logDirFailureChannel,
-          producerStateManager)
+          producerStateManager,
+          logPrefix)
```

Comment:
Could we add the new param to javadoc?

Reply:
Done. Addressed in cc8353fd9e2475e56488ade8d6f01a772830edc4.